### ISSUE TITLE
[Accessibility]: Radio input and others with the name "radio-group-0" are not grouped together

### ISSUE THREAD

**Issue by yashgarg1719** [carbon-design-system/carbon-charts#1409]
### Contact Details

yashgarg@kyndryl.com

### Environment

Browser, Automated testing tool and ruleset, Assistive technology used to verify

### Issue
Radio input and others with the name "radio-group-0" are not grouped together for `<ibm-radio>` component

### Expectation
Related sets of radio buttons or checkboxes should be programmatically grouped

### Screenshots

![image](https://user-images.githubusercontent.com/108860673/178191781-bface52f-be95-4d37-b89c-9e17b4e649c3.png)

### What WCAG 2.1 checkpoint does the issue violate?

IBM 1.3.1 Info and relationships

### Version

`@carbon/charts@0.41.79`
